ui/components/component-library/text/text.stories.tsx

```diff
@@ -143,6 +144,26 @@ export const FontStyleStory: StoryFn<typeof Text> = (args) => (
 
 FontStyleStory.storyName = 'Font Style';
 
+export const FontFamilyStory: StoryFn<typeof Text> = (args) => (
+  <Box
+    display={Display.Flex}
+    flexDirection={FlexDirection.Column}
+    gap={4}
+  >
+    <Text {...args} fontFamily={FontFamily.Default}>
+      Default Font (CentraNo1) - The quick brown fox jumps over the lazy dog
+    </Text>
+    <Text {...args} fontFamily={FontFamily.Accent}>
+      Accent Font (MMSans) - The quick brown fox jumps over the lazy dog
+    </Text>
+    <Text {...args} fontFamily={FontFamily.Hero}>
+      Hero Font (MMPoly) - The quick brown fox jumps over the lazy dog
+    </Text>
+  </Box>
+);
```

Comment:
Adding font family story in storybook

<img width="1441" alt="Screenshot 2025-05-13 at 1 23 01 PM" src="https://github.com/user-attachments/assets/88911fb2-6dd4-4798-b5e2-e568ae104cf6" />